README.md

```diff
@@ -69,7 +69,20 @@ The [sparrow container images](https://github.com/caas-team/sparrow/pkgs/contain
 
 ### Helm
 
-tbd
+Sparrow can be install via Helm chart. The chart is provided in multiple registries, so can use:
+
+```sh
+helm -n sparrow upgrade -i sparrow oci://ghcr.io/caas-team/charts/sparrow --version 1.0.0 --create-namespace
+```
+or
+
+```sh
+helm -n sparrow upgrade -i sparrow oci://mtr.devops.telekom.de/caas/charts/sparrow --version 1.0.0 --create-namespace
+```
```

Comment:
Typo fix and formatting:

```suggestion
Sparrow can be installed via Helm chart. The chart is provided in multiple registries, so you may choose the one that suits you best:

```sh
helm -n sparrow upgrade -i sparrow oci://ghcr.io/caas-team/charts/sparrow --version 1.0.0 --create-namespace
```

or

```sh
helm -n sparrow upgrade -i sparrow oci://mtr.devops.telekom.de/caas/charts/sparrow --version 1.0.0 --create-namespace
```
```
```suggestion
Sparrow can be installed via Helm chart. The chart is provided in multiple registries, so you may choose the one that suits you best:

```sh
helm -n sparrow upgrade -i sparrow oci://ghcr.io/caas-team/charts/sparrow --version 1.0.0 --create-namespace
```

or

```sh
helm -n sparrow upgrade -i sparrow oci://mtr.devops.telekom.de/caas/charts/sparrow --version 1.0.0 --create-namespace
```
```